tests/lean/simp_trace.lean.expected.out

```diff
@@ -19,7 +19,8 @@ Try this: simp only [g, Id.pure_eq]
       (let x := x;
       pure x)
 [Meta.Tactic.simp.rewrite] @Id.pure_eq:1000, pure x ==> x
-Try this: simp only [f1, modify, StateT.modifyGet, f2, StateT.bind, getThe, StateT.get, StateT.set]
+Try this: simp only [f1, modify, modifyGet, MonadStateOf.modifyGet, StateT.modifyGet, pure, f2, bind, StateT.bind, get,
+  getThe, MonadStateOf.get, StateT.get, set, StateT.set]
```

Comment:
Let's put a comment in the source for this one noting that prior to #2489 the `simp only` suggestion was incomplete, just in case there is ever a regression here and someone is tempted to revert this change in `.expected.out`.

Otherwise I think this is good to go!

Reply:
Done. Thanks again!